common/src/main/java/com/microsoft/identity/common/adal/internal/AuthenticationSettings.java

```diff
@@ -97,6 +115,30 @@ public void setSecretKey(byte[] rawKey) {
         mSecretKeyData.set(rawKey);
     }
 
+    /**
+     * set two raw bytes to derive secretKey to use in encrypt/decrypt. KeySpec
+     * algorithm is AES. used by broker only.
+     *
+     * @param secretKeys App related keys to use in encrypt/decrypt. Should contain two secret keys. 
+     */
+    public void setSecretKeys(final Map<String, byte[]> secretKeys) {
```

Comment:
This function already has a method called `setSecretKey` -- this is confusing.

This is for the broker only, right? I think this needs to be in another class or named differently. This is too similar to the existing method